registry/configuring_registry_storage/configuring-registry-storage-baremetal.adoc

```diff
@@ -12,6 +12,11 @@ include::modules/installation-registry-storage-config.adoc[leveloffset=+1]
 
 include::modules/registry-configuring-storage-baremetal.adoc[leveloffset=+1]
 
+[IMPORTANT]
```

Comment:
Would it be possible to include this in the module instead of at the assembly level? That way we could also get this included where the user is defining these options instead of at the end of the procedure.

Reply:
I agree it makes more sense in the module. I've added to step 1 (didn't want to add another note in step 2) and also included the xref to the supporting docs in the assembly so a user has link. Not ideal but should be better than having all the info after the procedure.

Reply:
This looks good to me!